ksqldb-rest-app/pom.xml

```diff
@@ -64,11 +64,6 @@
             <artifactId>kafka-json-serializer</artifactId>
         </dependency>
 
-        <dependency>
-            <groupId>io.confluent</groupId>
-            <artifactId>rest-utils</artifactId>
```

Comment:
Awesome to see this go! The main `pom.xml` still has `rest-utils-parent` specified as its parent, but that doesn't actually pull in rest-utils as a dependency, right? (Trying to build up my knowledge of Maven.)

Also, rest-utils is specified as an exclusion in assembly/package.xml files in various modules (e.g., https://github.com/confluentinc/ksql/blob/c7d1fff2e76872f9f3c670fe7db3ebfedc905204/ksqldb-rest-app/src/assembly/package.xml#L70). I think these exclusions are now obsolete? Not 100% sure.

Reply:
ack